scout/commands/delete/delete_command.py

```diff
@@ -21,7 +22,7 @@
     help="Restrict to cases with specified status",
 )
 @click.option("-older-than", type=click.INT, default=0, help="Older than (months)")
-@click.option("-rank-threshold", type=click.INT, default=5, help="With rank threshold lower than")
+@click.option("-rank-threshold", type=click.INT, default=10, help="With rank threshold lower than")
```

Comment:
@dnil what about this?

Reply:
I liked 5, only missed the default before.. 😄

Reply:
Ok. I'll leave it as it was, doesn't matter really.